### PR TITLE
[OSCD] Added a rule to detect possible Zerologon exploitation

### PR DESCRIPTION
**Pull request by aleqs4ndr**
For this issue [#574](https://github.com/Neo23x0/sigma/issues/574)

### CODE REVIEW

File: rules/windows/builtin/win_privesc_cve_2020_1472.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+title: 'Possible Zerologon (CVE-2020-1472) exploitation'
```
Comment (yugoslavskiy) [inline rules/windows/builtin/win_privesc_cve_2020_1472.yml:1]: there is an error in test: 

```
Rule rules/windows/builtin/win_privesc_cve_2020_1472.yml has a title that has not title capitalization. Words: 'exploitation'
```

Could you please fix it?

Comment (yugoslavskiy) [reply rules/windows/builtin/win_privesc_cve_2020_1472.yml]: Thank you!

**Comment by yugoslavskiy** [comment]
Thank you very much, @aleqs4ndr! Well done. It's good to see you participating again (:

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Comment by drdoc** [comment]
> Thank you very much, @aleqs4ndr! Well done. It's good to see you participating again (:

This rule will generate so many false-positive. Almost all 4742 events from changing domain controller's password, will match this rule.

And one more question to @aleqs4ndr, how are you planning to use "%DC-MACHINE-NAME%"?? How could we create sigma rules with list of domain controllers account names.

**Comment by aleqs4ndr** [comment]
> This rule will generate so many false-positive. Almost all 4742 events from changing domain controller's password, will match this rule.
> 
> And one more question to @aleqs4ndr, how are you planning to use "%DC-MACHINE-NAME%"?? How could we create sigma rules with list of domain controllers account names.

Hello @drdoc and thank you for your question.
The list of possible FPs are specified in the code.
Yes, it could generate FP but not so frequent from my perspective.
You can mitigate it via SIEM functionality (e.g built a timesheet of DC password expiring and use it against the rule) or manualy.

As for the lists of DC names - you may find another rules in Sigma project with logic relies on lists :)

**Pull request merged**